pixelpact/src/api.test.js

```diff
@@ -0,0 +1,79 @@
+import { buildFastify } from "./api.js";
+
+describe("", () => {
+  setupServer();
+  describe("POST /", () => {
+    it("returns 400 when actualHtml is missing", async () => {
+      const response = await app.inject({
+        method: "POST",
+        url: "/",
+        payload: {
+          expected: "ZXhwZWN0ZWQK",
+        },
+      });
+
+      expect(response.statusCode).toBe(400);
+
+      const body = JSON.parse(response.body);
+      expect(body.message).toBe(
+        "body must have required property 'actualHtml'"
+      );
+    });
+
+    it("returns 400 when expected is missing", async () => {
+      const response = await app.inject({
+        method: "POST",
+        url: "/",
+        payload: {
+          actualHtml: "<h1>Hello World</h1>",
+        },
+      });
+
+      expect(response.statusCode).toBe(400);
+
+      const body = JSON.parse(response.body);
+      expect(body.message).toBe("body must have required property 'expected'");
+    });
+
+    it("returns result when all parameters are provided", async () => {
+      const response = await app.inject({
+        method: "POST",
+        url: "/",
+        payload: {
+          actualHtml: "<h1>Hello World</h1>",
```

Comment:
Do I understand it correctly, that this value is anyway disregarded in the `mockSnapshot` function?

Reply:
yes, these tests only care about the API and are not interested in the actual snapshot logic